Language/Generics/Superbounded_types/well_bounded_A01_t03.dart

```diff
@@ -5,23 +5,30 @@
 /// @assertion Any use of a type `T` which is not well-bounded is a
 /// compile-time error.
 ///
-/// @description Checks that it is a compile-time error when a class with a not
-/// well-bounded type parameter is declared
+/// @description Checks that it is a compile-time error when a class has a type
+/// parameter bound which is a raw type that does not have simple bounds, and
+/// when a class has a type parameter bound which is malbounded.
 /// @author iarkh@unipro.ru
 
 class A<T extends A<T>> {}
 
+// Here B1 is a well-bounded type but the bound is a raw type A<A<dynamic>> that
```

Comment:
It's a little bit confusing to say that `A<A<dynamic>>` is a raw type: A raw type is a syntactic concept, and it refers to a term that has just a name (no actual type argument list), and it denotes a generic declaration. So `A` is a raw type. The _meaning_ of `A` is `A<A<dynamic>>`, and it is helpful to tell the reader that `A` has that meaning, but in this case I think it's more confusing than helpful. So maybe:

```suggestion
// Here `B1` is a well-bounded type, but the bound is a raw type `A` that
```